src/api/Annotations.js

```diff
@@ -70,14 +75,23 @@ export default class Annotations extends MarkerBasedApi {
         });
     }
 
-    updateAnnotation(
-        fileId: string,
+    updateAnnotation({
```

Comment:
unlikely: but can we confirm we're not using this directly in our parent apps? in case this would cause breaking changes

Reply:
Thanks for paying attention to that - I've checked and didn't found any usage